nevergrad/benchmark/experiments.py

```diff
@@ -1197,11 +1197,140 @@ def images(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+                    yield xp
+
+
+@registry.register
+def image_quality(seed: tp.Optional[int] = None, cross_val: bool=False) -> tp.Iterator[Experiment]:
+    """Optimizing images for quality:
+    TODO
+    """
+    seedg = create_seed_generator(seed)
+    optims: tp.List[tp.Any] = get_optimizers("structured_moo")
+    if default_optims is not None:
+        optims = default_optims
+    
+    # We optimize func_blur or func_brisque and check performance on func_iqa.
+    funcs: tp.List[ExperimentFunction] = [
+            imagesxp.Image(loss=imagesxp.imagelosses.Koncept512),
+            imagesxp.Image(loss=imagesxp.imagelosses.Blur),
+            imagesxp.Image(loss=imagesxp.imagelosses.Brisque),
+            ]
+    upper_bounds: tp.List[tp.Any] = [func(func.parametrization.sample().value) for func in funcs]
+    # TODO: add the proxy info in the parametrization.
+    for budget in [100 * 5 ** k for k in range(3)]:
+        for num_workers in [1]:
+            for algo in optims:
+                mofuncs = fbase.multi_experiments(funcs, upper_bounds=upper_bounds, no_crossval=[1, 2], pareto_size=16) if cross_val else [fbase.MultiExperiment(funcs, upper_bounds=upper_bounds)]
+                for func in mofuncs:
+                    xp = Experiment(func, algo, budget, num_workers=num_workers, seed=next(seedg))
+                    yield xp
+
+
+@registry.register
+def image_quality_cv(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Counterpart of image_quality with cross-validation."""
+    return image_quality(seed, cross_val=True)
+
+
+@registry.register
+def image_similarity_and_quality(seed: tp.Optional[int] = None, cross_val: bool=False) -> tp.Iterator[Experiment]:
+    """Optimizing images: artificial criterion for now."""
+    seedg = create_seed_generator(seed)
+    optims: tp.List[tp.Any] = get_optimizers("structured_moo")
+    if default_optims is not None:
+        optims = default_optims
+
+    
+    # 3 losses functions including 2 iqas.
+    func_iqa = imagesxp.Image(loss=imagesxp.imagelosses.Koncept512)
+    func_blur = imagesxp.Image(loss=imagesxp.imagelosses.Blur)
+    base_blur_value = func_blur(func_blur.parametrization.sample().value)
+    for func in [imagesxp.Image(loss=loss) for loss in imagelosses]:
+    
+        # Creating a reference value.
+        base_value = func(func.parametrization.sample().value)
+        mofuncs = fbase.multi_experiments([func, func_blur, func_iqa], upper_bounds=[base_value, base_blur_value, 100.], no_crossval=[1, 2], pareto_size=16) if cross_val else [
+                fbase.MultiExperiment([func, func_blur, func_iqa], upper_bounds=[base_value, base_blur_value, 100.])]
+        for budget in [100 * 5 ** k for k in range(3)]:
+            for num_workers in [1]:
+                for algo in optims:
+                    for mofunc in mofuncs:
+                        xp = Experiment(mofunc, algo, budget, num_workers=num_workers, seed=next(seedg))
```

Comment:
can this be factorized with `image_quality` ? There is a lot of code that looks really similar, and factorizing will help avoiding spending time modifying the same thing everywhere when we will need to (that always happen...)